crates/dprint/src/arg_parser.rs

```diff
@@ -580,13 +588,27 @@ impl ClapExtensions for clap::Command {
     self
       .arg(
         Arg::new("files")
+          .help("List of file patterns in quotes to format. This can be a subset of what is found in the config file.")
+          .num_args(1..),
+      )
+      .arg(
+        Arg::new("includes-override")
+          .long("includes-override")
+          .value_name("patterns")
           .help("List of file patterns in quotes to format. This overrides what is specified in the config file.")
           .num_args(1..),
       )
       .arg(
         Arg::new("excludes")
           .long("excludes")
           .value_name("patterns")
+          .help("List of file patterns or directories in quotes to exclude when formatting. This excludes in addition to what is found in the config file.")
+          .num_args(1..),
+      )
+      .arg(
+        Arg::new("excludes-override")
+          .long("excludes-override")
```

Comment:
Not entirely sure about this name. Another alternative: `--config-excludes` (but the problem that has is it's ambiguous where the base of the pattern is--relative to the cwd or config file?)

I think overall people won't be using this flag that much and I mostly kept it to maintain functionality with the previous versions, so maybe not a big deal whatever it's called.